metadata.rb

```diff
@@ -16,6 +16,7 @@
 supports 'arch'
 supports 'windows'
 
+depends 'apt'
```

Comment:
I'm not sure that `apt` should be added as a dependency for this cookbook. If it's needed for tests, then let's add it to `test/fixtures/policies/default.rb`:
```
cookbook 'apt'
```